fiftyone/operators/operations.py

```diff
@@ -299,35 +303,54 @@ def register_panel(
         on_change_extended_selection=None,
         allow_duplicates=False,
     ):
-        """Register a panel with the given name and lifecycle callbacks.
+        """Registers a panel with the given name and lifecycle callbacks.
 
         Args:
-            name: the name of the panel to register
-            icon (None): the icon to display in the panel tab
-            dark_icon (None): the icon to display in the panel tab in dark mode of app
-            light_icon (None): the icon to display in the panel tab in light mode of app
+            name: the name of the panel
+            label: the display name of the panel
+            icon (None): the icon to show in the panel's tab
+            light_icon (None): the icon to show in the panel's tab when the App
+                is in light mode
+            dark_icon (None): the icon to show in the panel's tab when the App
+                is in dark mode
             on_load (None): an operator to invoke when the panel is loaded
             on_unload (None): an operator to invoke when the panel is unloaded
-            on_change (None): an operator to invoke when the panel state changes
-            allow_duplicates (False): whether to allow multiple instances of the panel
+            on_change (None): an operator to invoke when the panel state
```

Comment:
These events are not supported here. Need to remove these:

```
            on_load (None): an operator to invoke when the panel is loaded
            on_unload (None): an operator to invoke when the panel is unloaded
            on_change (None): an operator to invoke when the panel state
                changes
            on_change_ctx (None): an operator to invoke when the panel
                execution context changes
            on_change_view (None): an operator to invoke when the current view
                changes
            on_change_dataset (None): an operator to invoke when the current
                dataset changes
            on_change_current_sample (None): an operator to invoke when the
                current sample changes
            on_change_selected (None): an operator to invoke when the current
                selected samples changes
            on_change_selected_labels (None): an operator to invoke when the
                current selected labels changes
            on_change_extended_selection (None): an operator to invoke when the
                current extended selection changes
```

Instead - these are supported as class methods only. They are enabled when the class method is defined.